Initial docker and docker-compose files

niwo opened the pull request:
Initial version of a docker setup for seven23_webapp.

Build it...
```
docker build -t seven23_webapp .
```
...and run it:
```
docker run -p 80:80 -ti seven23_webapp
```
Or directly with docker-compose:
```
docker-compose up
```

sebastienbarbier commented:
Thanks so much for this PR @niwo, great starting point 🙌.

One issue I had while testing is about nginx returning 404 error on refresh. Should be configured to return index.html to handle html5 urls 😎. Could be by creating a `nginx.conf` file in `src/www/config` as we have `.htaccess` for apache.
- [ ] Fix refresh returning 404

Might also be useful to update documentation with new docker stuff .
- [ ] Update documentation with docker references

niwo commented:
Thanks for your feedback!

- Fixed "refresh returning 404" using  nginx error_page config (sed within Dockerfile)
- Added a reference about the Docker setup within docs/build.rst

sebastienbarbier commented:
Let's merge and see how it goes 😁, looks awesome thanks again for taking the time to help.
